simple_docker.go

```diff
@@ -105,6 +105,10 @@ func install(arguments map[string]interface{}) (err error) {
 	return nil
 }
 
+func remove(arguments map[string]interface{}) (err error) {
+	return nil
+}
+
```

Comment:
Isn't the `remove` target only needed on Windows?

Reply:
Yeah, I added this to match `install()` but I don't think either are needed now that we're doing things in `platformTargets()`